tests/testProvenanceSetup.py

```diff
@@ -132,15 +136,15 @@ def testAddCmd(self):
         self.assertEquals(cmds[0][0], "recProv.py")
         self.assertEquals(cmds[0][1], "-v")
         self.assertEquals(cmds[0][2], "4")
-        
+
         self.setup.addWorkflowRecordCmd("dbingest.py", "-u ray".split(),
                                         "/usr/local/bin/dbingest.py")
```

Comment:
The code itself only manipulates lists and commands, and never tests were anything is.  It could be "/usr/foobar/woka/woka/dbingest.py" and it would work.   In practice, the code you gave above would be the correct way to invoke the API.  I don't see the harm in having the test the way it is.

I believe that the reason the code exists in this way is that provenance (at least when it was used in orchestration), records where the executables should be on the remote resource.  The ctrl_provenance code didn't execute on the remote resource, only from the launch platform.   Is this ideal?  I don't think it is, but it's what was done at the time.